### DISCUSSION TITLE
React Native

### DISCUSSION THREAD

**Discussion by ghost** [remirror/remirror#779] [Q&A] [answered]
Does remirror support react native?

**Comment by ifiokjr** [comment, marked as answer]
React Native support is not yet generally available.

**Comment by PabraiVentures** [comment]
How about now? Is there any date where RN is expected to be supported if not? @ifiokjr

**Reply by strdr4605** [reply]
@ocavue, @ronnyroeller, @ankon, any news on ReactNative support?

**Reply by ankon** [reply]
I don't think anyone is actively working on ReactNative support right now. But, if you need it, why not help out? :)

**Reply by strdr4605** [reply]
Sure, I need to discuss with my team, and if we decide to go with @remirror, we can help with ReactNative support.

**Reply by psdewar** [reply]
@strdr4605 any updates?

@ifiokjr I saw references to remirror__extension-react-native-bridge and remirror/react-native. Why were they deprecated?

**Reply by strdr4605** [reply]
@psdewar2, I did not have time to investigate remirror ReactNative, I might try to do some POC in Q3.

**Reply by ronnyroeller** [reply]
> I saw references to remirror__extension-react-native-bridge and remirror/react-native. Why were they deprecated?

We deprecated those because we couldn't find a maintainer who has deep enough understanding about ReactNative. We decided it's best to communicate this transparently to the community by deprecating.
Having said that, if you know somebody who would maintain ReactNative support: we could surely discuss to bring them back. Let us know :)
